designer/client/src/components/graph/node-modal/editors/expression/SqlEditor.tsx

```diff
@@ -88,35 +88,18 @@ function useAliasUsageHighlight(token = "alias") {
 }
 
 const SqlEditor: SimpleEditor<Props> = (props: Props) => {
-    const { expressionObj, onValueChange, className, formatter, ...passProps } = props;
-    const sqlFormatter = formatter == null ? typeFormatters[FormatterType.Sql] : formatter;
-
-    const valueChange = useCallback(
-        (value: string) => {
-            const encoded = sqlFormatter.encode(value);
-            if (encoded !== value) {
-                return onValueChange(encoded);
-            }
-        },
-        [onValueChange, sqlFormatter],
-    );
-
+    const { expressionObj, onValueChange, className, ...passProps } = props;
     const value = useMemo(
         () => ({
-            expression: sqlFormatter.decode(expressionObj.expression.trim()),
-            language: ExpressionLang.SQL,
+            expression: expressionObj.expression,
+            language: ExpressionLang.SqlSpELTemplate,
```

Comment:
This language is only for code formatting, isn't it? Otherwise, why did it work if we didn't have support for SQL on BE side? If so, please keep the old formatting language name, we don't need to spread this temporary `SqlSpELTemplate` in many places